tests/publiccloud/create_aistack_env.pm

```diff
@@ -74,9 +74,9 @@ sub install_dependency_components {
     my $ing_ver = get_var('ING_VERSION');
 
     # Add Ingress Controller to open-webui endpoint
-    assert_script_run("helm repo add $ingress_repo");
-    assert_script_run("helm repo update");
-    assert_script_run("helm upgrade --install ingress-nginx ingress-nginx/ingress-nginx --namespace ingress-nginx --set controller.service.type=ClusterIP --version $ing_ver --create-namespace", timeout => 120);
+    #assert_script_run("helm repo add $ingress_repo");
+    #assert_script_run("helm repo update");
+#assert_script_run("helm upgrade --install ingress-nginx ingress-nginx/ingress-nginx --namespace ingress-nginx --set controller.service.type=ClusterIP --version $ing_ver --create-namespace", timeout => 120);
```

Comment:
Please, for formatting and tidy questions, any comment `#` should have _at least 1 space_ separting text.
Moreover, comments should be _aligned_, then L79 in the same column of previouses.

Reply:
Fixed